.gitattributes

```diff
@@ -0,0 +1,63 @@
+###############################################################################
```

Comment:
Overall the program works great, I haven't found even a single bug, the view is clean and functional. We could ship this game today :)
Almost all my remarks have to do with OO style of programming and MVVM pattern. A couple of style and algorithm improvements to be made.